website/server/controllers/api-v3/groups.js

```diff
@@ -344,7 +345,14 @@ api.getGroups = {
     }
 
     if (req.query.member) {
-      filters._id = { $in: user.guilds };
+      // Includes 'guilds' nor 'publicGuilds' type.
```

Comment:
can we move the check to `Group.getGroups ` here https://github.com/HabitRPG/habitica/blob/develop/website/server/models/group.js#L279 so that the tavern is never returned when that method is used instead of just in this route?

Reply:
_Move_ the check or keep it here and _add_ the check to `Group.getGroups` as well?

Reply:
I'd move the check